projects/vdk-plugins/vdk-server/src/vdk/plugin/server/server_plugin.py

```diff
@@ -71,9 +84,9 @@ def server(install, uninstall, status):
     else:
```

Comment:
Can you wrap all the code in one block?

at the moment errors in check status will still be silent.